css/base.css

```diff
@@ -155,6 +155,19 @@ dt div {
   margin-top: 0;
 }
 
+:is(aside, div).example .example-title {
```

Comment:
This seems to be our first time using `:is` for these docs; flagging to make sure we are okay with its [level of browser support](https://caniuse.com/css-matches-pseudo)

Reply:
`:is` is being used in TR space so I am comfortable with this